underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -162,9 +165,18 @@ public HdfsUnderFileSystem(AlluxioURI ufsUri, UnderFileSystemConfiguration conf,
           Class.forName(HDFS_EC_CODEC_REGISTRY_CLASS);
         } catch (ClassNotFoundException e) {
           LOG.warn("Cannot initialize HDFS EC CodecRegistry. "
-              + "HDFS EC will not be supported: {}", e.toString());
+              + "HDFS EC will not be supported:", e);
         }
       }
+
+      try {
+        Class.forName(JAVAX_WS_RS_CORE_MEDIA_TYPE);
```

Comment:
do you mind adding a comment for why this is needed?

Reply:
was this the class being loaded by a different classloader?

Reply:
yes, if this class is not loaded here, it will be later loaded by the system classloader from alluxio server jar, but will be called by a class loaded with extension classloader.